### PR TITLE
fix(boards): Remove duplicate VID/PID from boards.txt and add default CDC/JTAG board

### PR DESCRIPTION
**Pull request by me-no-dev**
Adds a hidden board with the default VID/PID of USB CDC/JTAG in order to always show that in IDE when ESP32 CDC/JTAG device is connected and removes all other mentions of that VID/PID

### CODE REVIEW

**Comment by me-no-dev** [comment]
@P-R-O-C-H-Y we need to enforce that this VID/PID combo is not added to other boards

File: boards.txt [hunk 1]
```diff
@@ -38,6 +38,8 @@ esp32_family.name=ESP32 Family Device
 esp32_family.hide=true
 esp32_family.vid.0=0x303a
 esp32_family.pid.0=0x1001
+esp32_family.upload_port.0.vid=0x303a
+esp32_family.upload_port.0.pid=0x1001
```
Comment (me-no-dev) [inline boards.txt:42]: @ladyada in order for discovery to work properly with IDE2 and CLI, it is necessary to add the `.upload_port` lines. Maybe someone there can update your board definitions?

cc: @UnexpectedMaker

**Review by P-R-O-C-H-Y — APPROVED** [review]

File: boards.txt (hunk 1 above)
Comment (ladyada) [reply boards.txt:42]: thanks
@hathach can you add to your ToDo list :)
@me-no-dev is there a date you need this by?

Comment (me-no-dev) [reply boards.txt:42]: @ladyada we plan on pushing bugfix releases fairly often, so it's not a hurry.

Comment (hathach) [reply boards.txt:42]: @ladyada sure, I will follow this up

**Comment by UnexpectedMaker** [comment]
Hey @me-no-dev - Thanks for pinging me as well...

So you mean we need to add this to each of our boards, using the same PID/VID as the board?
```
um_bling.name=UM BLING
um_bling.vid.0=0x303a
um_bling.pid.0=0x817F
um_bling.upload_port.0.vid=0x303a
um_bling.upload_port.0.pid=0x817F
```

**Comment by me-no-dev** [comment]
> So you mean we need to add this to each of our boards, using the same PID/VID as the board?

Yes. First two lines are for IDE1 and second lines are for IDE2 and CLI

**Comment by UnexpectedMaker** [comment]
Wicked, thanks, doing a PR now :)

**Pull request merged**